binlog_streamer.go

```diff
@@ -35,37 +36,37 @@ type BinlogStreamer struct {
 	eventListeners []func([]DMLEvent) error
 }
 
-func (s *BinlogStreamer) Initialize() error {
```

Comment:
What's the reasoning for removing the `Initialize` method?

Reply:
It's a legacy of the original Ghostferry design. This method doesn't do much and just complicates the API. Anything it does can be done directly in the `Run` method, or within the `Ferry.NewBinlogStreamer` methods.